.gitattributes

```
@@ -0,0 +1,3 @@
+# https://docs.github.com/en/github/administering-a-repository/customizing-how-changed-files-appear-on-github
+deps/metadata.json linguist-generated=true
+src/services/* linguist-generated=true
```

I missed this auto-generated file:
```suggestion
src/AWSServices.jl linguist-generated=true
src/services/* linguist-generated=true
```